services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -153,7 +167,7 @@
     @Field("radiuses") String radiuses,
     @Field("steps") Boolean steps,
     @Field("bearings") String bearings,
-    @Query("avoid_maneuver_radius") Integer avoidManeuverRadius,
+    @Field("avoid_maneuver_radius") Integer avoidManeuverRadius,
```

Comment:
Unrelated change?

Reply:
Yes, I can extract it out to a separate commit.